database/migrations/2020_05_11_100000_create_waivers_table.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Tipoff\Locations\Models\Location;
+
+class CreateWaiversTable extends Migration
```

Comment:
I renamed the table to just waivers. It also must be migrated before the signatures table so I changed the date. The signatures table will need to have a relationship with a waiver instead of a location.

Reply:
@drewroberts I don't really understand this.

`Waiver` belongs to `Signature`.
and `Signature` belongs to `Location`?

Reply:
Signature currently belongs to a Location. THat will be changed in the near future so that Signature belongs to a Waiver

Reply:
I'm going to make some adjustments on the Waiver too. I want a location to have multiple Waivers, but only 1 active waiver at a time. Need to think through that a little more. It will allow us to have records of what version of the waiver a participant signed.

Reply:
@drewroberts but signatures table doesn't have `location_id` column currently so I don't think Signature currently belongs to a Location.